src/docs/Updates/Roadmap/index.mdx

```diff
@@ -4,54 +4,77 @@ order: 2
 description: 'Discover what we are working on right now'
 ---
 
-## Q2 2020 (from 01/04/2020 to 30/06/2020)
+## Q3 2020 (from 01/07/2020 to 30/09/2020)
 
-### Sprint 1 (Week 22 to 24)
+### Sprint 1 (Week 28 to 30)
 
-**Add new patterns on Mozaic:**
+**Add new patterns:**
 
-- ~~Form patterns~~
+- ~~Tooltip~~
+- ~~Breadcrumb~~
 
 **Mozaic-Vue:**
 
-- ~~Add Pagination~~
-- ~~Improve homepage and storybook UI~~
+- ~~Add Mozaic-Vue link in Mozaic's homepage~~
 
-### Sprint 2 (Week 24 to 26)
+### Sprint 2 (Week 30 to 32)
 
-- ~~Tabs~~
+**Updates and fixes:**
 
-### Sprint 3 (Week 26 to 28)
+- ~~Update File uploader documentation and add error cases~~
 
-**Add new patterns:**
+**Mozaic-Figma:**
 
-- ~~Modal~~
-- ~~Accordion~~
+Add:
 
-### Sprint 4 (Week 28 to 30)
+- ~~Colors library~~
+- ~~Links library~~
+- ~~Icons library~~
+
+Fixes:
+
+- ~~Grids use now the right values~~
+- ~~Fonts: Change line-heights by the right values (instead of auto values)~~
+
+### Sprint 3 (Week 32 to 36)
+
+**Website:**
+
+- Refactor menu
+- Refactor table of content
+- Improve pattern page header
+- Add VueJS link inside the pattern pages
 
 **Add new patterns:**
 
-- Tooltip
-- Breadcrumb
+- Layers
+- Heroes
 
 **Mozaic-Vue:**
 
-- Add Mozaic-Vue link in Mozaic's website
+- Add some existing patterns of Mozaic
 
-### Sprint 5 (Week 30 to 32)
+**Mozaic-Figma:**
```

Comment:
can we rephrase like that?
⬇️

Mozaic-Figma add ons:

- All form patterns
- Flags
- Dividers
- Breadcrumb
- Pagination
- Tabs
- Tooltips